ElementX/Sources/Other/Extensions/URL.swift

```diff
@@ -28,8 +28,17 @@ extension URL {
     /// The URL of the primary app group container.
     static var appGroupContainerDirectory: URL {
         guard let url = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: InfoPlistReader.main.appGroupIdentifier) else {
-            fatalError("Should always be able to retrieve the container directory")
+            MXLog.error("Application Group unavailable, falling back to the application folder")
+            // Browserstack doesn't properly handle AppGroup entitlements so this fails, presumably because of the resigning happening on their side
+            // Try using the normal app folder instead of the app group
+            // https://www.browserstack.com/docs/app-automate/appium/troubleshooting/entitlements-error
+            guard let url = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first?.deletingLastPathComponent().deletingLastPathComponent() else {
+                fatalError("Should always be able to retrieve the app directory")
+            }
+            
+            return url
```

Comment:
We discussed that this fallback isn't going to cause any harm elsewhere.

Small suggestion:
```suggestion
            return URL.applicationSupportDirectory.deletingLastPathComponent().deletingLastPathComponent()
```
Also, how come we don't use the directory provided and instead go up 2 levels?

Reply:
Because I didn't want to add 1 `Library/Application Support` layer to the existing one

Reply:
I'm not sure I understand the suggestion: `applicationSupportBaseDirectory` is using the `appGroupContainerDirectory` internally 🤔

Reply:
> Because I didn't want to add 1 `Library/Application Support` layer to the existing one

Oh sorry yes this overall property is the bast directory I see. The suggestion is purely to use the non-failable `URL.applicationSupportDirectory` instead of `urls(for: .applicationSupportDirectory, in:) which returns an optional.

Reply:
Ah sorry, I mistook `applicationSupportDirectory` for our own `applicationSupportBaseDirectory`